doc/getting_started/contributing.rst

```diff
@@ -113,7 +143,7 @@ Building Documentation
 ======================
 
 Building the package documentation requires a working Sphinx installation,
-and latex to build the pdf. As of this writing we are using sphinx 1.2.
+and latex to build the pdf. As of this writing we are using sphinx 8.0.2.
```

Comment:
wow that's a big jump...

Reply:
yeah, i didn't actually update anything either, documentation just hadn't been updated since then